client/lib/analytics/index.js

```diff
@@ -344,6 +354,31 @@ const analytics = {
 		}
 	},
 
+	// Lucky Orange tracking
+	luckyOrange: {
+		initialized: false,
```

Comment:
@fditrapani and @ockham I question if it should belong in Redux since it never seems to impact the rendered UI.

on that note, I wonder if we need it at all. do we have to specifically track if it has been loaded? shouldn't there be some indication that it has? could we not just make a function which returns whether or not it has been loaded instead of storing that value?

```js
// I made up the name of the global obviously
export const hasLuckyOrange = () => undefined !== window.loGtVS3dj;
```